onebusaway-enterprise-webapp/src/main/java/org/onebusaway/enterprise/webapp/actions/api/StopForIdAction.java

```diff
@@ -54,6 +55,9 @@
 
   @Autowired
   private TransitDataService _transitDataService;
+	
+  @Autowired
+  private TripsConfigurationServiceClientFileImpl tConfigurationServiceClientFileImpl;
```

Comment:
why is this called tConfigurationServiceClientFileImpl in some locations and _ tConfigurationServiceClientFileImpl in others?  Please review the OBA coding conventions